install/linux/usr/share/odemis/sim/meteor-tfs2-sim.odm.yaml

```diff
@@ -0,0 +1,238 @@
+# For now, this is esssentially just a FM microscope, as the FIB part is handled
+# separately by the SEM
+
+METEOR-Sim: {
+    class: Microscope,
+    role: meteor,
+    children: [
+        "Stage",
+        "Meteor Stage",
+        "Optical Objective",
+        "Optical Focus",
+        "Light Source",
+        "Filter Wheel",
+        "Camera",
+        "EBeam",
+    ],
+}
+
+"SEM": {
+    class: simsem.SimSEM,
```

Comment:
I think that in this case it'd make more sense to use the xt_client, with the assumption that the xt_adapter simulator is running on the localhost. This way, you'd show the way to activate raw coordinates too.